tor/onionfile/onion_file_test.go

```diff
@@ -0,0 +1,95 @@
+package onionfile
+
+import (
+	"bytes"
+	"io"
+	"io/ioutil"
+	"path/filepath"
+	"testing"
+
+	"github.com/stretchr/testify/require"
+)
+
+var (
+	privateKey = []byte("RSA1024 hide_me_plz")
+)
+
+type MockEncrypter struct{}
+
+func (m MockEncrypter) EncryptPayloadToWriter(_ bytes.Buffer, _ io.Writer,
+	_ []byte) error {
+
+	return nil
+}
+
+func (m MockEncrypter) DecryptPayloadFromReader(_ io.Reader, _ []byte) ([]byte,
+	error) {
+
+	return privateKey, nil
+}
+
+// TestOnionFile tests that the File implementation of the OnionStore
+// interface behaves as expected.
+func TestOnionFile(t *testing.T) {
+	t.Parallel()
+
+	tempDir, err := ioutil.TempDir("", "onion_store")
+	require.NoError(t, err, "unable to create temp dir")
+
```

Comment:
need `defer os.RemoveAll(tempDir)`

Reply:
Instead, we should start using `t.TempDir()` everywhere in tests, that automatically removes the directories.